### DISCUSSION TITLE
Database only backups not encrypted

### DISCUSSION THREAD

**Discussion by aronvdl** [spatie/laravel-backup#1422] [Q&A] [answered]
Hi, When I run a complete backup through spatie, the zip-file is encrypted, like I configured. But when I make a Database only backup, the zip-file is not encrypted.. Is there an aditional setting needed?
Kind regards, Aron
btw I love your product!

**Comment by PaolaRuby** [comment, marked as answer]
You have to give all the information, the authors cannot guess or use psychic powers to know which package version you use, php version, etc, it's the only way someone can give you an answer

it is verified that the package does encrypt when it is used `--only-db`
https://github.com/spatie/laravel-backup/blob/4fe55e63559a4ccdbfa497a303b00751e9af4abd/tests/Commands/BackupCommandTest.php#L380-L395

**Reply by PaolaRuby** [reply]
`spatie/laravel-backup 6.16.5` has the test too, that means it is working correctly
https://github.com/spatie/laravel-backup/blob/e7324472d4384ef1dc089400723ff01745497625/tests/Commands/BackupCommandTest.php#L376-L391

**Reply by PaolaRuby** [reply]
@aronvdl I am using cron job and it works correctly, maybe try [Laravel Task Scheduling](https://laravel.com/docs/8.x/scheduling)

**Reply by aronvdl** [reply]
Yeah!! That worked! Thanks a lot!

**Comment by aronvdl** [comment]
Hello Paola, You are right! Here some information:
Laravel v8.65
PHP 7.4.20
spatie/db-dumper                   2.21.1  Dump databases
spatie/laravel-backup              6.16.5  A Laravel package to backup your application
spatie/temporary-directory         1.3.0   Easily create, use and destroy temporary directories
Ubuntu 18.04

**Comment by aronvdl** [comment]
Ok, I found out when the problem with none encrypted files occurs. When I backup trough a shell command, the files are created, stored and encrypted as they should. When the command is executed as a cron-job, the zip-files are created, stored, but NOT encrypted.

Any idea what could cause that?

The cron job command is: 
cd /var/www/vhosts/xxx.nl/httpdocs && /opt/plesk/php/7.4/bin/php 'artisan' backup:run

(with xxx.nl as my domain)
